packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -334,6 +386,7 @@ export class CheckoutOrderSummary extends PureComponent {
                 { this.renderItems() }
                 { this.renderCmsBlock() }
                 { this.renderTotals() }
+                { isMobile && this.renderDiscountCode() }
```

Comment:
Im confused. Out here u check if mobile and then render. Why in this case in renderDiscountCode function it again checks if mobile and renders depending on it.